### ISSUE TITLE
Autocrafting unit attached to an ME network stops functioning on chunk unload until refreshed

### ISSUE THREAD

**Issue by T5UN4M1** [ReikaKalseki/Reika_Mods_Issues#1966]
The autocrafting unit only works when I change the number of items in the ME network.

For example, if I want to produce sticks from planks, in sustain or continuous mode, I have to take a stack of sticks and put them one by one in the ME system to update the number of sticks and the AU will produce sticks. (1 craft each time I drop a stick in the terminal)

**Comment by ReikaKalseki** [comment]
Related to #1931?

**Comment by ReikaKalseki** [comment]
Also, the Autocrafter will sit idle if it has no work to do, and crafting from the ME system will change its contents - obviously - and should therefore keep updating whether it has work.

**Comment by T5UN4M1** [comment]
No, it's not a connection issue, the autocrafter is connected to the network, but it simply refuses to do any work unless the amount of input / output items is changed in the network in which case it will produce "1 craft" (unless you do it too quickly)

Here is a short clip that shows the problem : https://www.youtube.com/watch?v=sx_ubrUjKcU

Also, there is that "input limit" thing that I always leave on default value (unlimited) , I don't know what that thing does but I hope that it isn't the cause of the problem.

**Comment by ReikaKalseki** [comment]
> it simply refuses to do any work unless the amount of input / output items is changed in the network in which case it will produce "1 craft" (unless you do it too quickly)

That is my point. That one craft will subtract some ingredients from the network, thus satisfying that condition, and priming it for further crafts.

**Comment by uniacke1** [comment]
It works as you describe, but it seems that the update it creates does not cause it to re-craft. 

e.g. if I have it in sustain for a stack of redstone dust, and use my last stack, it will craft 9 from a block via the recipe. The re-addition of those 9 to the ME system does not make it then add another 9 until it is over 64 again.

Same with continuous. But it works fine it something else triggers the update by consuming said item.

**Comment by ReikaKalseki** [comment]
Might be an order of operations problem...

**Comment by voidsong-dragonfly** [comment]
I have a similar problem, but mine seems to be caused by chunk unloading/reloading while the machine is running.

**Comment by tiny-maple** [comment]
I have a similar problem too as @Silfryi, where the machine will fail to autocraft when chunk is unloaded until interacted with the item in the ME system.
